front-end/src/services/teacherService.js

```diff
@@ -1,11 +1,12 @@
 import API from 'API';
 
-export const getAllTeacher = async () => {
-  const { data } = await API.get('/api/teachers');
-  return data;
-};
-
-export const getTeacherById = async (id) => {
-  const { data } = await API.get(`/api/teachers/${id}`);
-  return data;
+export const teacherService = {
```

Comment:
Why did you refactor this? As I know on a project we use the left side of these changes.

Reply:
It is very strange because I did not refactor that. I checked it by using git lens and git lens showed me that Nadia refactored code.